delta/rdf/src/main/scala/ch/epfl/bluebrain/nexus/delta/rdf/jsonld/context/ExtendedJsonLdContext.scala

```diff
@@ -0,0 +1,38 @@
+package ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context
+
+import ch.epfl.bluebrain.nexus.delta.rdf.implicits._
+import io.circe.Json
+import io.circe.syntax._
+import org.apache.jena.iri.IRI
+
+/**
+  * A Json-LD context that has been inspected to obtain its relevant fields.
+  *
+ * @param value          the value of the @context key
+  * @param base           the IRI value of the @base key if present
+  * @param vocab          the IRI value of the @vocab key if present
+  * @param aliases        the @context aliases used to compact or shorten keys/values
+  * @param prefixMappings the @context prefix mappings used to form CURIES
+  */
+final case class ExtendedJsonLdContext(
+    value: Json,
+    base: Option[IRI] = None,
+    vocab: Option[IRI] = None,
+    aliases: Map[String, IRI] = Map.empty,
+    prefixMappings: Map[String, IRI] = Map.empty
+) extends JsonLdContext {
+  type This = ExtendedJsonLdContext
+
+  lazy val aliasesInv: Map[IRI, String] = aliases.map { case (prefix, iri) => iri -> prefix }
```

Comment:
this could hide duplicates, is that acceptable?

Reply:
It could. In case of duplicates it will choose the first sorted alphabetically. So I'll change the implementation